Sources/Routing/Routing/RouterNode.swift

```diff
@@ -2,27 +2,30 @@
 final class RouterNode<Output> {
     /// Kind of node
     var value: Data
-
+    
```

Comment:
Could you remove all these whitespace changes? It makes it hard to see what really changed.

Reply:
Sure thing - did not even notice!

Reply:
Just updated!